opencensus/trace/ext/grpc/client_interceptor.py

```diff
@@ -86,6 +87,16 @@ def _start_client_span(self, client_call_details):
             attribute_key=attributes_helper.GRPC_ATTRIBUTES.get(GRPC_METHOD),
             attribute_value=str(client_call_details.method))
 
+        # Add user defined auxiliary attributes to span
+        if type(self.aux_attributes) is dict:
```

Comment:
Nit, but you might want `isinstance(dict)` here so e.g. `defaultdict`s work here too.